### DISCUSSION TITLE
怎么配置才能访问自己的后端？

### DISCUSSION THREAD

**Discussion by wushenhaoyu** [HalseySpicy/Geeker-Admin#404] [Q&A]
我改了.env里的开发环境跨域代理为我的服务器地址
然后把port1变为空了，接着去访问，然后返回500没反应

**Comment by elarbor** [comment]
![image](https://github.com/user-attachments/assets/4a807aac-cd8f-4c08-a07d-00726e3cac92)
![image](https://github.com/user-attachments/assets/e7239417-ee6b-4f73-920a-a71c30648705)
